components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { Text, Box, SimpleGrid, GridItem, Flex } from "@chakra-ui/react";
+import DaysOfWeekButton from "./DayOfTheWeekButton";
+import LinkButton from "../LinkButton/LinkButton";
+
+export default function DaysOfTheWeekContainer ({ setNumberOfDays, onSaveEvent }) {
+
+  // initial state for days of the week has info if it's selected or not (instead of having 2 separate states)
+  const [ daysOfTheWeek, setDaysOfTheWeek ] = useState([
+    {
+      id: 0,
```

Comment:
More a question than a suggestion for change, @OlaShabalina :
what/how is `id` variable used?
I couldn't see an example of its use in this PR...

Reply:
@hqtan I was thinking of storing data in a short form by using Id instead of Day string in future if we want to show the breakdown by days. Doesn't matter for the current setup. Should I remove it?

Reply:
Thx for explanation @OlaShabalina :bow: 
we might have need for this var soon! Let's leave it for now :+1: